src/pages/Contact/Contact.js

```diff
@@ -32,6 +31,7 @@ const Contact = ({ data }) => {
           <h3>{ data.section_3.section_title }</h3>
           <div className="contact__col" dangerouslySetInnerHTML={{ __html: data.section_3.section_body.left }}></div>
           <div className="contact__col" dangerouslySetInnerHTML={{ __html: data.section_3.section_body.right }}></div>
+          <div className="contact__col"><iframe src={ data.section_1.section_body.right } className="contact__map--mobile" title="Google Maps" frameBorder="0" aria-hidden="false" tabIndex="0"></iframe></div>
```

Comment:
Please double check all the spacing in the file, i see a couple things that could make this a bit easier to read

```suggestion
          <div className="contact__col">
            <iframe src={ data.section_1.section_body.right } className="contact__map--mobile" title="Google Maps" frameBorder="0" aria-hidden="false" tabIndex="0"></iframe>
          </div>
```